docs/source/en/model_doc/trajectory_transformer.mdx

```diff
@@ -0,0 +1,49 @@
+<!--Copyright 2022 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# TrajectoryTransformer
```

Comment:
Given that it's a brand new model with an API that is very different from the original implementation, I think it would significantly help usage if we could add some in-depth examples here, either in the markdown or as a link to a colab notebook.